selfdrive/car/ford/carstate.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, CP):
     if CP.transmissionType == TransmissionType.automatic:
       self.shifter_values = can_define.dv["Gear_Shift_by_Wire_FD1"]["TrnRng_D_RqGsm"]
 
+    self.vehicle_sensors_valid = False
```

Comment:
we can make this a carState field either now or in the future when we start to check other quality flags